PWGDQ/Core/VarManager.h

```diff
@@ -963,6 +968,18 @@ void VarManager::FillTrack(T const& track, float* values)
     values[kMass] = track.mass();
   }
 
+  // Derived quantities which can be computed based on already filled variables
+  FillTrackDerived(values);
+}
+
+template <uint32_t fillMap, typename U, typename T>
+void VarManager::FillTrackMC(const U& mcStack, T const& track, float* values)
+{
+  if (!values) {
+    values = fgValues;
+  }
+
+  // Quantities based on the mc particle table
   if constexpr ((fillMap & ParticleMC) > 0) {
```

Comment:
Since now this function has only one purpose (fill quantities for the ParticleMC), the template argument fillMap is not really necessary at the moment. I suggest to drop it from the function definition, unless you have plans in the short term to include more bits in the fillMap which would apply for this function.